xline/src/storage/kvstore.rs

```diff
@@ -550,12 +551,12 @@ impl KvStoreBackend {
             version: new_rev.version,
             ..KeyValue::default()
         };
-        let prev = self.db.insert(new_rev.as_revision(), kv.clone());
+        let _prev = self.db.insert(new_rev.as_revision(), kv.clone());
```

Comment:
What if another put request takes place between line 541 and line 554? Is the previous value correct in this case?

Reply:
this key is in spec poll when sync, another put request of the same key is conflict, it cannot execute and sync, so this case is impossible